core/lib/zksync_core/src/api_server/web3/namespaces/en.rs

```diff
@@ -156,4 +156,16 @@ impl EnNamespace {
         };
         Ok(config)
     }
+
+    #[tracing::instrument(skip(self))]
+    pub async fn whitelisted_tokens_for_aa_impl(&self) -> Result<Vec<Address>, Web3Error> {
+        Ok(self
+            .state
+            .tx_sender
+            .0
+            .whitelisted_tokens_for_aa_cache
+            .read()
+            .await
+            .clone())
```

Comment:
Nit: may make sense to extract this getter to a method, since it's used in several places.